lib/resque/failure/sentry.rb

```diff
@@ -16,26 +17,26 @@ module Failure
     #   Resque::Failure.backend = Resque::Failure::Multiple
     #
     class Sentry < Base
-
-      def self.logger
-        @logger
+      class << self
+        attr_reader :logger
       end
 
-      def self.logger=(value)
-        @logger = value
+      class << self
+        attr_writer :logger
       end
 
       def save
         options = {}
         options[:logger] = self.class.logger if self.class.logger
```

Comment:
is `options[:logger]` deprecated in the `sentry-ruby`? It seems like it doesn't get used https://github.com/getsentry/sentry-ruby/blob/master/sentry-ruby/lib/sentry/scope.rb#L78-L92